src/dds/rs-dds-internal-data.cpp

```diff
@@ -0,0 +1,178 @@
+                                     { RS2_FORMAT_Y16, RS2_STREAM_INFRARED, 2 } },
+                                   []() { return std::make_shared< y12i_to_y16y16_mipi >(); } } );
+        }
+        else
+        {
+            tmp = processing_block_factory::create_pbf_vector< yuy2_converter >( RS2_FORMAT_YUYV,
+                                                                                 target_formats( RS2_FORMAT_YUYV ),
+                                                                                 RS2_STREAM_COLOR );
+            for( auto & it : tmp )
+                factories.push_back( std::move( it ) );
+            factories.push_back( processing_block_factory::create_id_pbf( RS2_FORMAT_RAW16, RS2_STREAM_COLOR ) );
+            factories.push_back( { { { RS2_FORMAT_Y12I } },
+                                   { { RS2_FORMAT_Y16, RS2_STREAM_INFRARED, 1 },
+                                     { RS2_FORMAT_Y16, RS2_STREAM_INFRARED, 2 } },
+                                   []() { return std::make_shared< y12i_to_y16y16 >(); } } );
+        }
+        tmp = processing_block_factory::create_pbf_vector< uyvy_converter >(
+                               RS2_FORMAT_UYVY,
+                               target_formats( RS2_FORMAT_UYVY ),
+                               RS2_STREAM_INFRARED );
+        for( auto & it : tmp )
+            factories.push_back( std::move( it ) );
+        factories.push_back( processing_block_factory::create_id_pbf( RS2_FORMAT_Y8, RS2_STREAM_INFRARED, 1 ) );
+        factories.push_back( processing_block_factory::create_id_pbf( RS2_FORMAT_Z16, RS2_STREAM_DEPTH ) );
+        factories.push_back( { { { RS2_FORMAT_W10 } },
+                               { { RS2_FORMAT_RAW10, RS2_STREAM_INFRARED, 1 } },
+                               []() { return std::make_shared< w10_converter >( RS2_FORMAT_RAW10 ); } } );
+        factories.push_back( { { { RS2_FORMAT_W10 } },
+                               { { RS2_FORMAT_Y10BPACK, RS2_STREAM_INFRARED, 1 } },
+                               []() { return std::make_shared< w10_converter >( RS2_FORMAT_Y10BPACK ); } } );
+        factories.push_back( { { { RS2_FORMAT_Y8I } },
+                               { { RS2_FORMAT_Y8, RS2_STREAM_INFRARED, 1 },
+                                 { RS2_FORMAT_Y8, RS2_STREAM_INFRARED, 2 } },
+                               []() { return std::make_shared< y8i_to_y8y8 >(); } } );
+
+        // Motion convertion is done on the camera side.
+        // Intrinsics table is being read from the camera flash and it is too much to set here, this file is ment as a
+        // temporary solution, the camera should send all this data in a designated topic.
+        factories.push_back( { { { RS2_FORMAT_MOTION_XYZ32F, RS2_STREAM_ACCEL } },
+                               { { RS2_FORMAT_MOTION_XYZ32F, RS2_STREAM_ACCEL } },
+                               []() { return std::make_shared< identity_processing_block >(); } } );
```

Comment:
Since the default is to have identity, no need to specifically mention IMU -- but this will be removed in next PR, right?